cli/azd/pkg/infra/provisioning/bicep/bicep_provider_test.go

```diff
@@ -81,7 +81,7 @@ func TestBicepPlanPrompt(t *testing.T) {
 	mockContext.CommandRunner.When(func(args exec.RunArgs, command string) bool {
 		return strings.Contains(args.Cmd, "bicep") && args.Args[0] == "--version"
 	}).Respond(exec.RunResult{
-		Stdout: "Bicep CLI version 0.12.40 (41892bd0fb)",
+		Stdout: fmt.Sprintf("Bicep CLI version %s (abcdef0123)", bicep.BicepVersion.String()),
```

Comment:
Without this change the tests end up taking longer because the bicep provider thinks it needs to update to the latest bicep version, which takes some time.